### DISCUSSION TITLE
How to import exported rules/accounts/etc?

### DISCUSSION THREAD

**Discussion by nagyv** [firefly-iii/firefly-iii#4787] [Firefly III]
I am running Firefly III version 5.5.11.

**Description**

I exported account and rules setups as described in https://docs.firefly-iii.org/firefly-iii/exporting-data/

How can I import it back?

**Bonus points**

- I have read the FAQ at https://bit.ly/FF3-FAQ
- I was able to replicate the issue on the demo site.

**Comment by JC5** [comment]
Not easily I'm afraid. You can parse the CSV into queries though.

**Comment by nagyv** [comment]
I just found https://github.com/pmatseykanets/artisan-io that might be able to solve the import. Unfortunately, I don't know the PHP/Laravel ecosystem at all. Trying to install `artisan-io` gives

```
composer require pmatseykanets/artisan-io
Using version ^1.0 for pmatseykanets/artisan-io
./composer.json has been updated
Running composer update pmatseykanets/artisan-io
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires pmatseykanets/artisan-io ^1.0 -> satisfiable by pmatseykanets/artisan-io[v1.0.0].
    - pmatseykanets/artisan-io v1.0.0 requires illuminate/support ~5.1 -> found illuminate/support[v5.1.1, ..., v5.8.36] but these were not loaded, likely because it conflicts with another require.
```

I understand that this is a compatibility issue, but don't understand what the 2nd line mean. Is there any easy fix for the above?

Would it be possible to solve the import with `artisan-io` or not because of related fields/Models?

**Comment by JC5** [comment]
You should install 3.0.0, not 1.0.0.
